### DISCUSSION TITLE
Implement MQTT functionality in HACS addon

### DISCUSSION THREAD

**Discussion by djoenez** [serkri/SmartEVSE-3#193] [General]
Hello,

I absolutely love the MQTT addition to the EVSE, i was wondering if it is in the planning to add this to the HACS addon? 
another way of working for me would be to rename all the sensors/switches that HACS makes, but this seems a bit cumbersome (and error prone).

**Comment by dingo35** [comment]
No the HACS addon uses the REST API, and can be used concurrent with the MQTT interface.

If you go into HomeAssistant/Settings/Devices&Services/Devices and you select the MQTT SmartEVSE device, you can add all sensors to a dashboard with one press of the button, same for diagnosis entities and mode selector entities.
